prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs

```diff
@@ -109,6 +109,47 @@ public override IEnumerable<License> SeedData
                     new License { Code = 85, Weight = 4, Name = "Non-practicing registration" },
                     new License { Code = 86, Weight = 4, Name = "Temporary registration" },
 
+
+                    //College of Oral Health Professionals
+                    // Dental Assistant
```

Comment:
Please check ordering of licence classes versus https://oralhealthbc.ca/register-renew/registrant-classifications/certified-dental-assistant/

Reply:
The last 3 licence classes have been re-ordered from Leanne in the excel spreadsheet. It is different that the order in web site.